samples/hello_world/src/main.c

```diff
@@ -4,10 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/kernel.h>
+#include <stdio.h>
 
 int main(void)
 {
-	printk("Hello World! %s\n", CONFIG_BOARD);
```

Comment:
I think showing the board name is a useful feature, especially if you imagine this might become part of a sysbuild project running on multiple cores, each exposed through a single USB CDC device with multiple ports.

Reply:
maybe we could recommend https://github.com/zephyrproject-rtos/zephyr/pull/61141 for such cases (provided it's modified to print board name)